tests/file_system_cache_test.py

```diff
@@ -101,14 +101,15 @@ def test_least_recently_accessed_file(self):
       cache = FileSystemCache(tmpdir, max_cache_size_bytes=6)
       cache.put("foo", b"bar")
       cache.put("second", b"two")
-      #Added sleeps because these operations were executing too fast that the access time could not be captured properly
+      # Sleep because otherwise these operations execute too fast and
+      # the access time isn't captured properly.
       time.sleep(1)
       cache.get("foo")
       cache.put("third", b"the")
       self.assertEqual(cache.get("foo"), b"bar")
       self.assertEqual(cache.get("second"), None)
 
-  @jtu.ignore_warning(message="This compilation file is larger than the size of the cache")
+  @jtu.ignore_warning(message=("Cache value of size {} is larger than the max cache size of {}").format(len(b"bar"), 2))
```

Comment:
You don't need to use a format string here at all, you can just hardcode the actual warning that the test produces (something like "Cache value of size 3 is larger than the max cache size of 2"). This makes it easier to read.